src/routes/Home/index.tsx

```diff
@@ -287,6 +290,17 @@ const Home = ({ videoId }: { videoId: string }) => {
     utils.history.push(`/edit/${videoId}`);
   };
 
+  const descText = selectedVideo ? (selectedVideo as any).snippet.description : 'None';
+  const descPreview = descText.substring(0, 150) + ' ...';
+  const extraContent = (
+    <div>
+      <p className="extra-content">{descText}</p>
+    </div>
+  );
+
+  const moreButtonName = readMore ? 'Collapse' : 'Read More';
+  const isLandingPage = selectedVideo ? selectedVideo.id === '_ok27SPHhwA' : false;
```

Comment:
This is okay until we go public, however the `_ok27SPHhwA` video is an actual video so this is really a hack that overrides that video.  We can work on a better method later.